app/assets/stylesheets/application.css

```diff
@@ -163,20 +156,87 @@ dark = <<-DARK
 	--color-mobile-story-comments-bg: #101010;
 	--color-mobile-story-comments-bubble-fill: #282828;
 	--color-mobile-story-comments-bubble-fill-zero: #1c1c1c;
-DARK
-%>
-
-:root {
-	<%= light %>
-}
-@media (prefers-color-scheme: dark) {
-	html.color-scheme-system {
-		<%= dark %>
-	}
+  }
 }
+
+
 html.color-scheme-dark {
-	<%= dark %>
-}
+	/* main monochrome */
```

Comment:
This is duplicated. I couldn't find a good way to reuse the variables and their values. 

On a side note, It will be nice to deprecate the user setting of light/dark mode and use the system settings, as most operating systems and browsers support it now. This will simplify the codebase as well.

Reply:
Did you look at [light-dark](https://12daysofweb.dev/2024/css-light-dark/)? I linked it earlier, maybe it would work for removing the duplication.

I don’t want to remove the user setting. I can’t claim to understand it, but people are feel very strongly about having it per-site.

Reply:
Yeah, I spent a minute reading the MDN page. In the [example section](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark#example) under the heading "CSS" they show moving all the variables into `:root`. Then an example of making a section of the page light mode, but rather than having a `.light` class to force the color scheme, we'd have `html.color-scheme-dark { color-scheme: dark; } html.color-scheme-light { color-scheme: light; }`.

I think that would solve this duplication.